sklearn/manifold/locally_linear.py

```diff
@@ -289,7 +289,14 @@ def locally_linear_embedding(
         raise ValueError("unrecognized method '%s'" % method)
 
     nbrs = NearestNeighbors(n_neighbors=n_neighbors + 1, n_jobs=n_jobs)
+
```

Comment:
No need to add a new line here. In general only change the minimal/required amount of things to make the review process easier and more focussed.

Reply:
Fixed